besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -63,7 +63,7 @@
 import org.hyperledger.besu.testutil.TestClock;
 import org.hyperledger.enclave.testutil.EnclaveKeyConfiguration;
 import org.hyperledger.enclave.testutil.EnclaveTestHarness;
-import org.hyperledger.enclave.testutil.OrionTestHarnessFactory;
+import org.hyperledger.enclave.testutil.TesseraTestHarnessFactory;
```

Comment:
Why this change from Orino to Tessera?

Reply:
Orion is end-of-life and Tessera is the new replacement. This was easier than updating the Orion harness.